server/database/config/dbBuild.sql

```diff
@@ -1,77 +0,0 @@
-BEGIN;
-
-    DROP SCHEMA public CASCADE;
```

Comment:
why delete `dbBuild.sql`. if this pull request merge this file will delete from the master